engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -341,4 +371,32 @@ - (int64_t)lastAppModificationTime {
   return [fileDate timeIntervalSince1970];
 }
 
+- (UISceneConfiguration*)application:(UIApplication*)application
```

Comment:
sanity check: this chunk is the same as previous pr right?

Reply:
It's close.  I also switched to a dedicated scene delegate which was feedback on the other pr.